src/west/__init__.py

```diff
@@ -1,3 +1,6 @@
-# Copyright (c) 2020, Nordic Semiconductor ASA
+# Copyright (c) 2021 Nordic Semiconductor ASA
 
-# nothing here
+import logging
+
+# https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library
+logging.getLogger('west').addHandler(logging.NullHandler())
```

Comment:
> This results in just the ProjectCommandLogHandler output being shown

Is that on stdout ? Many CI engines and maybe others color stderr differently, extremely useful.

Reply:
> Is that on stdout ? Many CI engines and maybe others color stderr differently, extremely useful.

No, it's on stderr.